frappe/core/doctype/system_settings/system_settings.json

```diff
@@ -485,7 +486,14 @@
    "fieldtype": "Select",
    "label": "First Day of the Week",
    "options": "Sunday\nMonday\nTuesday\nWednesday\nThursday\nFriday\nSaturday"
-  }
+  },
+  {
+    "default": "1200",
+    "fieldname": "reset_password_link_expiry_seconds",
+    "fieldtype": "Int",
+    "label": "Reset Password Link Expiry Seconds",
```

Comment:
"Password Reset Expiry (in Seconds)"

Reply:
Changed it to `Reset Password Link Expiry (in Seconds)`